src/app/doctor/handlers/handlers.go

```diff
@@ -24,6 +26,26 @@ func (h *Handler) CreateDoctorHandler(c echo.Context) error {
 		return utils.CreateEchoResponse(c, http.StatusBadRequest, nil)
 	}
 
+	if err := h.validator.Struct(doctorRequest); err != nil {
+		if strings.Contains(err.Error(), "email") {
+			return utils.CreateEchoResponse(
+				c,
+				http.StatusBadRequest,
+				response.ErrorResponse{Reason: "email is invalid"},
+			)
+		}
+
+		if strings.Contains(err.Error(), "password") {
+			return utils.CreateEchoResponse(
+				c,
+				http.StatusBadRequest,
+				response.ErrorResponse{
+					Reason: "password must have at least 8 characters",
+				},
+			)
+		}

```

Comment:
Why the error reasons not declared in a variable then return the echo response once?

Reply:
done https://github.com/ALTERA-CAPSTON-41/backend/pull/65/commits/5b09e3601784691dfab73275a3470fe55137059d